modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -120,7 +122,14 @@ object FormInputEV {
     def render(p: Props[EV, A], s: State): VdomNode = {
 
       val validationError: js.UndefOr[Label] =
-        s.errors.map(e => Label(e.toList.mkString(","))).orUndefined
+        s.errors
+          .map(e =>
+            Label(content = e.toList.mkString(","),
+                  clazz = p.errorClazz,
+                  pointing = p.errorPointing
+            )(^.position.absolute)
```

Comment:
Should we not force `position.absolute` and define it in the `errorClazz` as necessary?

Reply:
Surprisingly it doesn't work. I was fiddling with this for a while but couldn't make it work. I'll leave it as an exercise :D